### PR TITLE
add function to multiply

### PR DESCRIPTION
**Pull request by paulo-santana**
**kkkkkkkkkkk daora descrição massa** 
zuera po

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -4,10 +4,15 @@ int sum(int x, int y) {
...
 }
 
+int mul(int x, int y) {
+	return x * y;
+}
+
 int main(void) {
-	int x = 1;
-	int y = 2;
+	int x = 5;
```
Comment (hde-oliv) [inline main.c:12]: Why do you changed that?

**Review by hde-oliv — COMMENTED** [review]
Fix the values, please!

File: main.c (hunk 1 above)
Comment (paulo-santana) [reply main.c:12]: Because, why not?

PS: did*

Comment (hde-oliv) [reply main.c:12]: Fix it please.

Comment (paulo-santana) [reply main.c]: fixed

**Comment by hde-oliv** [comment]
Thanks for your contribution!
Fix: #1

**Pull request merged**